package.json

```diff
@@ -39,7 +39,8 @@
       "lodash.template": ">=4.5.0",
       "meow": "8.1.2",
       "mixin-deep": ">=1.3.2",
-      "prettier": "^1.15.3",
+      "prettier": "^3.2.5",
```

Comment:
**Prettier 1** -> **Prettier 3** is a major version upgrade. ⚠️ 

Before doing such change you must check what breaking changes there were between v1 and v3 before upgrading.
 Need to make sure it doesn't cause any issues.
Code might still compile with no problem. 

Problem is when you create a new PR, that changes just 1 code line, you also get 100s of lines of non-sense code styling ([Example](https://github.com/SAP/luigi/pull/3745/files#diff-4bfc2a7ab3f5e1777cc276378c9b396cbf38bae43de4418980a50412c86f9b70) ) . This makes code  reviews difficult as you can't see easily what code changes were made by Developer and which by prettier commit hook. 

Need to assess first what "damage" the new prettier version causes by running prettier on all JS/TS relevant code files on the repo, and see how many file changes it makes. 

Too see the effects of the new prettier version, run:
`npm run full-code-quality-prettier`

on root folder. And check with `git status` , how many files were affected. 

_Note_: This is not  just svelte-specific comment, as prettier affects HTML, JS and TS files in our repo.

Reply:
You're definitely right I shouldn't introduce breaking change that would affect many files in this PR - I downgraded both prettier and svelte plugin to version one. After running 'full-code-quality-prettier' command only svelte files are affected (having your code preferences in mind)

Reply:
I checked all the options, but this plugin (or Prettier itself) doesn't allow to ignore specific language feature (like '@const' tag) - the incorrect plugin behavior was fixed in version 2.6.0:
https://github.com/sveltejs/prettier-plugin-svelte/blob/master/CHANGELOG.md#260

That's why I think we have to use comments for ignoring formatting just like I proposed and we can remove them when we decide to upgrade Prettier at least to version 2 (same goes for the plugin)

Reply:
ok. then we stick to the comments 👍